Formula/kde-extra-cmake-modules.rb

```diff
@@ -3,6 +3,7 @@ class KdeExtraCmakeModules < Formula
   homepage "https://api.kde.org/frameworks/extra-cmake-modules/html/index.html"
   url "https://download.kde.org/stable/frameworks/5.73/extra-cmake-modules-5.73.0.tar.xz"
   sha256 "c5e3ef0253f7d5ab3adf9185950e34fd620a3d5baaf3bcc15892f971fc3274c4"
+  license "BSD-3-Clause"
```

Comment:
Does that BSD 3 license subsume both `LICENSES/BSD-2-Clause.txt` and `LICENSES/MIT.txt`?